### PR TITLE
Send operation data when polling last_operation after provision

### PR DESCRIPTION
**Pull request by nodo**
Similarly to what we have done in https://github.com/openservicebrokerapi/osb-checker/pull/25/files, we need to pass operation query params when polling last_operation during provision

Signed-off-by: Andrea Nodari <anodari@pivotal.io>

### CODE REVIEW

**Comment by mattmcneeney** [comment]
LGTM

@duglin for the other one?

**Comment by nodo** [comment]
Hey @mattmcneeney @duglin , I have just noticed that this PR is pending for a while? do you have any updates on this? Is there anything I can do to speed up the process?

**Comment by leonwanghui** [comment]
+1

**Comment by mattmcneeney** [comment]
@openservicebrokerapi/osbapi-pmc one more LGTM needed please!

**Comment by mattmcneeney** [comment]
Sorry for the delay @nodo

**Comment by zhongyi-zhang** [comment]
This is a nice TODO item I commented in the test framework. I'll help rebase and resolve conflicts.

**Comment by zhongyi-zhang** [comment]
Rebase the change in https://github.com/openservicebrokerapi/osb-checker/pull/39. @leonwanghui @norshtein could you have a review?

File: 2.13/tests/test/test.js [hunk 1]
```diff
@@ -445,48 +453,51 @@ function testDeprovision(instanceId, queryStrings, isAsync){
...
-                    if (err) callback(err);
-                    var message = validateJsonSchema(res.body, lastOperationSchema);
-                    if (message != "") callback(new Error(message));
-                    lastOperationState = res.body.state;
-                })
             setTimeout(function() {
+                count++;
+                console.log(count.toString() + 'th polling last operation...');
+                var url = '/v2/service_instances/' + instanceId + '/last_operation';
+                if (lastOperationName) url += '?operation=' + lastOperationName;
```
Comment (leonwanghui) [inline 2.13/tests/test/test.js]: I think it would be better if you could assert if `lastOperationName == ""`

Comment (zhongyi-zhang) [reply 2.13/tests/test/test.js]: From the [spec](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#parameters), adding this query string or not depends on whether the last operation returns an `operation` in response body, right?

Comment (leonwanghui) [reply 2.13/tests/test/test.js]: Yes

**Comment by norshtein** [comment]
LGTM

**Pull request merged**